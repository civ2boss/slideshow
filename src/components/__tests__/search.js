import React from 'react';
import renderer from 'react-test-renderer';

import Search from '../Search';

test('Search renders without crashing', () => {
  const component = renderer.create(<Search />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
