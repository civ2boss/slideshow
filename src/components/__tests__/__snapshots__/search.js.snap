// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders without crashing 1`] = `
<form
  className="search"
  onSubmit={undefined}
>
  <input
    className="search-box"
    name="search"
    onChange={undefined}
    placeholder="Searchâ€¦"
    type="search"
    value={undefined}
  />
</form>
`;
